index.bs

```diff
@@ -855,7 +848,10 @@ Instances of <code>ReadableStreamReader</code> are created with the internal slo
   1. If IsReadableStreamReader(*this*) is *false*, throw a *TypeError* exception.
   1. If *this*@[[ownerReadableStream]] is *undefined*, return *undefined*.
   1. If *this*@[[readRequests]] is not empty, throw a *TypeError* exception.
-  1. Perform CloseReadableStreamReader(*this*).
+  1. If *this*@[[ownerReadableStream]]@[[state]] is `"readable"`, reject *this*@[[closedPromise]] with a *TypeError* exception.
+  1. Otherwise, let *this*@[[closedPromise]] be a new promise rejected with a *TypeError* exception.
```

Comment:
Ah, I see. This new rejected promise and the TypeError at L851 are representing "already released". Then, looks good.